src/test/resources/application.yml

```diff
@@ -34,4 +34,16 @@ graphql:
 flyway:
   datasources:
     default:
-      locations: classpath:migrations
\ No newline at end of file
+      locations: classpath:migrations
+
+email:
+  voting:
+    recipient: somebody@email.com
+    subjectTemplate: How are you today %s ?
+    bodyTemplate: |
+      Hi %s:
+
+      Looking forward to hearing from you in group %s.
+
+      Cheers
+      Admin
```

Comment:
This is not working:

- [ ] Check these properties are used
- [ ] Move them to aws.mail
- [ ] In the future this has to be taken from a template-dir by template-key and use a proper template engine